cmd/boots/dhcp.go

```diff
@@ -97,6 +104,8 @@ func (d dhcpHandler) serveDHCP(w dhcp4.ReplyWriter, req *dhcp4.Packet) {
 		return
 	}
 	span.End()
+	j.HttpServerFQDN = d.httpServerFQDN
```

Comment:
nitpick: uppercase `j.HttpServerFQDN` to `j.HTTPServerFQDN`?

Reply:
ah yes, thank you. update.